### ISSUE TITLE
Usage directions hard to find

### ISSUE THREAD

**Issue by obfischer** [alainfrisch/nostalgy#68]
The usage directions for Nostalgy available at http://alain.frisch.fr/info/ABOUT_NOSTALGY.html are hard to find, but should be easy to find.

Users starting from https://addons.thunderbird.net/de/thunderbird/addon/nostalgy/ do not have a change to find them. Starting from this page they can install Nostalgy or go to the homepage of Nostalgy at http://alain.frisch.fr/soft_mozilla.html. But this page does not contain a link to the usage directions. 

Therefore I would like to have the usage directions as part of the information available via Github. 

I volunteer for this task.

**Comment by MLopez-Ibanez** [comment]
Hi @obfischer Obfisher, could you send a pull request to my fork? https://github.com/MLopez-Ibanez/nostalgy

Or if you attach the markdown text here, I'll merge it and make a pull request.

**Comment by caugner** [comment]
I agree that it would be nice to add the Usage directions to the repository and replace the link in the *Nostalgy Preferences* accordingly.

**Comment by caugner** [comment]
Actually the *Usage directions* already in this repository (see [`about.xhml`](https://github.com/nostalgy/nostalgy/blob/master/chrome/content/about.xhtml)):

https://github.com/nostalgy/nostalgy/blob/7dc4f78de515cf8dbb58306585a17356603e5fa0/chrome/content/about.xhtml#L20

**Comment by caugner** [comment]
@obfischer Would it be enough to link to this file from the `README.md`?
